.gitignore

```diff
@@ -1,2 +1,3 @@
 sysroot
 build
+.vscode
```

Comment:
Please put editor-specific paths in your own global git ignore file, rather than in repository .gitignore files.